tests/TreeHouse/Queue/Tests/Amqp/Driver/AbstractDriverConnectionTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace TreeHouse\Queue\Tests\Amqp\Driver;
+
+use PHPUnit_Framework_Assert as Assert;
+use TreeHouse\Queue\Amqp\ConnectionInterface;
+
+abstract class AbstractDriverConnectionTest extends \PHPUnit_Framework_TestCase
+{
+    use DriverTestTrait;
+
+    /**
+     * @test
+     */
+    public function it_can_Creation()
```

Comment:
strange method name?

<img src="http://img.memecdn.com/if-you-cant-win-an-argument-correct-their-grammar-instead_o_3923309.webp"/>